### DISCUSSION TITLE
Option to download beatmap packs through osu!direct

### DISCUSSION THREAD

**Discussion by apollo-dw** [ppy/osu#14328] [Ideas]
downloading packs on the website in a compressed format and then needing to extract them seems bothersome, being able to press one button to queue downloads to osu!direct would be Cool

**Comment by iSimpFor** [comment]
maybe a pack could be a header while looking like a multiplayer playlist(except that all maps of a mapset are combined into a entry on the list) 

the beatmap packs could also have its seperate toolbar button next to osu!direct itself

for packs with a medal attached, maybe there could be a medal shown on the header(grayed out if not obtained, solid if obtained)

one question, would DMCA'ed mapsets be hidden and show up only if you actually have the mapset? or would DMCA'ed mapsets be removed from the pack entirely?
